### DISCUSSION TITLE
How to disable search bar?

### DISCUSSION THREAD

**Discussion by k0va1** [scalar/scalar#2869] [Q&A]
Hi! I couldn't find a property in configuration to disable search. Is it possible right now?

**Comment by antlio** [comment]
hey @k0va1, thanks for reaching out! it is not possible through our current configuration. do you mind sharing a bit more why would you like to disable it? did you encounter anything that led you that way? thanks again!

**Comment by k0va1** [comment]
hey @antlio! sure, i can explain. i'm using scalar inside my project as part of the page where there're a lot of other elements. i have my own global search with CMD+k binding so these two searches conflict with each other :( i realize that it's not a trivial usage of the tool but it'd be nice to have an ability to toggle search like `hideDarkModeToggle`

**Reply by antlio** [reply]
awesome thanks for sharing, that makes sense! i'll share that use case and come back to you asap!

**Comment by alixodendhal** [comment]
I also have this issue... But I don't have a planned solution. 
The issue is mainly: How should we handle docusuarus search along with Scalar Search? 

Most of us want a site-wide search. I guess the only option is to add a site-wide plugin and hide/drop the Scalar search?

I _do_ like the Scalar search, but I don't know how I should handle the conflict. It seems I would need to pick whichever search acts sitewide. 

@k0va1 what search sitewide search plugin are you using?

related -> https://github.com/scalar/scalar/discussions/1390
